ToDoApp/Backend/src/resolvers/todo/todos.spec.js

```diff
@@ -41,75 +81,58 @@ describe('mutate', () => {
             });
             expect(res.errors).toHaveLength(1);
         });
-        it('logIn with correct credentials returns token-String', async () => {
-            const res = await mutate({
-                mutation: LOGIN,
-                variables: {
-                    email: "your@email.com",
-                    password: "password"
-                }
-            });
-            expect(res.data.login).toBeDefined();
-        });
     });
     describe('give user is loggedIn', () => {
         it('adds a todo', async () => {
-            const res_allTodos=await query ({
-                    query: GET_ALL_TODOS
-            });
             const res = await mutate({
                 mutation: CREATE_TODO,
                 variables: {
                     message: "neues Todo",
                     token: logInToken
                 }
             });
-            expect(res.data.addTodo).toHaveLength(res_allTodos.data.allTodos.length+1);
-            expect(res.data.addTodo[res.data.addTodo.length-1].message).toEqual("neues Todo");
-
+            expect(res.data.addTodo[0].message).toEqual("neues Todo");
         });
         describe('Modifying Todos', () => {
+         /**
```

Comment:
Same

Reply:
Did you know that you can run only specific test cases with `jest -t "test case title"`?